lib/order.rb

```diff
@@ -0,0 +1,66 @@
+
+require 'csv'
+require_relative 'customer'
+
+# Class that holds information on order
+class Order
+  # Constants for order's status
+  VALID_STATUS = [:pending, :paid, :processing, :shipped, :complete]
+  
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+  
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    
+    # Check for valid order status
+    raise ArgumentError, "Status is bogus" unless VALID_STATUS.include?(@fulfillment_status)
```

Comment:
It's helpful for the user (and for you when debugging) to include the bad argument in the message of your `ArgumentError`.  

```suggestion
    raise ArgumentError, "Status #{fulfullment_status} is bogus" unless VALID_STATUS.include?(@fulfillment_status)
```